src/types/ut_types/UtilsTests.cpp

```diff
@@ -190,9 +190,9 @@ void UtilsTests::TestColorFromXTermColor()
     _VerifyXTermColorResult(L"yellow", RGB(255, 255, 0));
     _VerifyXTermColorResult(L"yellow3", RGB(205, 205, 0));
     _VerifyXTermColorResult(L"wheat", RGB(245, 222, 179));
-    _VerifyXTermColorResult(L"wheat4", RGB(139, 126, 102));
+    _VerifyXTermColorResult(L"wheat4", RGB(139, 126, 101));

```

Comment:
The formula itself is pretty accurate. But the rounding method is not ideal. Do we have a rounding method that can round to the nearest integer? For example the real value here is actually ~101.8, so the result should be 102. With the correct rounding method I think the test case can remain intact.